misc/test_ping

```diff
@@ -5,20 +5,37 @@ RESULT_LINES=/tmp/result_lines.txt
 MONO_LOG=/tmp/mono-log.txt
 MANIFEST=./module_manifest.json
 REPORT_FILE=/tmp/report.txt
+status=pass
 
 echo Baseline ping test report | tee -a $MONO_LOG
+ls -l 
+if [ ! -f /scans/startup.pcap ]; then
+    exit 1
+fi
 
-# This is a check that the startup.pcap file exists and is proper.
-echo
-ls -l /scans/startup.pcap
 echo %% $(tcpdump -n -r /scans/startup.pcap | wc -l) packets captured. | tee -a $MONO_LOG
 echo
-tcpdump -en -r /scans/startup.pcap ip
-echo
-tcpdump -ven -r /scans/startup.pcap port 67 | fgrep "length 1: Request"
-echo
-tcpdump -ven -r /scans/startup.pcap port 67 | fgrep "length 1: ACK"
-echo
+
+STATIC_IP=`jq -r .static_ip /config/device/module_config.json`
+
+if [ -n "$STATIC_IP" ]; then
+    echo RESULT skip base.startup.request static ip| tee -a $RESULT_LINES
+    echo RESULT skip base.startup.ack static ip| tee -a $RESULT_LINES
+else
+    req=pass
+    ack=pass
+    # This is a check that the startup.pcap file exists and is proper.
```

Comment:
maybe this should be above? (Just the basic ip capture part) rather than just checking for existance.  Just having this one line would be sufficient to replace the entire if (since it'll fail if the file is missing)